assets/scripts/app/App.jsx

```diff
@@ -3,6 +3,9 @@ import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { IntlProvider } from 'react-intl'
 import NOTIFICATION from '../../../app/data/notification.json'
+import HTML5Backend from 'react-dnd-html5-backend'
+import { DragDropContext } from 'react-dnd'
+import flow from 'lodash/flow'
```

Comment:
Nitpicky thing (here and elsewhere) I prefer to have imports from external libraries up top, sorted roughly in order of "importance" (whatever that looks like; ideally, it's more or less a stable order across different modules), followed by internal modules. The internal modules kind-of have an order of importance (or at least, grouped by similar locations) but that hasn't been as well adhered to.